app/src/renderer/components/staking/PageBond.vue

```diff
@@ -237,10 +237,19 @@ export default {
           })
           this.$router.push("/staking")
         } catch (err) {
-          this.$store.commit("notifyError", {
-            title: "Error While Bonding Atoms",
-            body: err.message
-          })
+          let errData = err.message.split("\n")[5]
```

Comment:
This will probably be used more often (like with sending coins). Can we implement this somehow in `send.js` like changing the error message there?

Reply:
I'm trying to figure out which types of errors are encountered while transferring tokens